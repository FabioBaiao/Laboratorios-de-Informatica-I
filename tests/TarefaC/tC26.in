abcc
aaac
aaaC
aaDC
0 3 E
SSADAALALES