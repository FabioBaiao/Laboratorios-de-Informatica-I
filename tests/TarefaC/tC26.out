3 1
3 0
INCOMPLETO
